index.rst

```diff
@@ -1,61 +1,1351 @@
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+Here is that architecture in diagram form:
 
-   Feel free to delete this instructional comment.
+.. figure:: /_static/science-platform.png
+   :name: High-level Science Platform architecture
 
-:tocdepth: 1
+   An overview of a Rubin Science Platform deployment.
+   This shows the major aspects of the Science Platform but omits considerable detail, including most supporting services, the identity management store, and some details of the Gafaelfawr architecture.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+As discussed in DMTN-234_, there is no single Rubin Science Platform.
+There are multiple deployments of the Science Platform at different sites with different users and different configurations.
+With respect to the identity management system, these differ primarily in the choice of the first two components.
 
-.. sectnum::
+General access deployments of the Science Platform use CILogon_ as the source of user authentication, and COmanage_ as the repository of identity information.
+Here is the architecture for general access deployments, expanding the identity management portion and simplifying the rest of the Science Platform to a single protected service:
 
-.. TODO: Delete the note below before merging new content to the main branch.
+.. _CILogon: https://www.cilogon.org/
+.. _COmanage: https://www.incommon.org/software/comanage/
 
-.. note::
+.. figure:: /_static/general-access.png
+   :name: General access identity management architecture
+
+   Detail of the components for identity management for a general access deployment of the Science Platform.
+   The Science Platform aspects and services are represented here by a single service to make the diagram simpler.
+
+Restricted access deployments use either GitHub or a local `OpenID Connect`_ authentication provider as the source of user authentication; and one of GitHub, a local LDAP server, or the OpenID Connect authentication provider as the source of identity information.
+
+.. _OpenID Connect: https://openid.net/specs/openid-connect-core-1_0.html
+
+A restricted access deployment that uses GitHub looks essentially identical to the first architecture diagram with GitHub as the identity provider.
+One that uses OpenID Connect is similar, but will likely separate the identity provider box into an OpenID Connect provider and an LDAP server that will be queried for metadata.
+
+Identity management
+===================
+
+The identity management component of the system authenticates the user and maps that authentication to identity information.
+
+In general access deployments, authentication is done via the OpenID Connect protocol using CILogon, which in turn supports SAML and identity federations as well as other identity providers such as GitHub and Google.
```

Comment:
Instead of differentiation on general or restricted access, just differentiate on the provider choice. "In our federated identity deployments" or "in our CILogon deployments" etc. There's no point baking in the general/restricted differentiation in a technology note